### PR TITLE
Redesign contributing page

### LINKED ISSUE #401: Redesign: Contribution Page
This task is to implement a redesigned contributing page as shown in the mock up images below. This is our current contributing page and where you can find the content for the page http://www.theodinproject.com/contributing

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] Check out our [styleguide](https://staging-odin-project.herokuapp.com/styleguide)
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch  

If you have any questions or need clarification about anything related to this task please contact the maintainers in our [contributing channel on Gitter](https://gitter.im/TheOdinProject/Contributing)

### Contributing redesign mock up
contribution

### PR DESCRIPTION
**Pull request by wanyamaman**
closes #401

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "hstore"
```
Comment (arun1595) [inline db/schema.rb]: Why do we need a `hstore` extension?

File: app/views/static_pages/contributing.html.erb [hunk 2]
```diff
@@ -1,68 +1,107 @@
...
+          </div>
+        </div>
+        <br/>
+        <div class="row">
+          <div class="col-md-8 offset-md-2 mb-5">
+            <p class="text-muted">
+              Instructions on setting up your local development environment can be found in the <%= link_to 'GitHub repo', github_link, target: '__blank' %>.
+            </p>
+            <p class="text-muted">
+              <em>NOTE: When choosing an issue to work on, please check out the <%= link_to '#contributing', chat_link, class: 'gitter-chat-link', target: '_blank' %> channel on our Gitter chat, and let everyone know what you are working on.  This is important, as somebody else may have already done some work on the issue - in which case they would probably love to have your help!</em>
```
Comment (arun1595) [inline app/views/static_pages/contributing.html.erb]: The contributing link should be pointing to https://gitter.im/TheOdinProject/Contributing

File: db/schema.rb (hunk 1 above)
Comment (KevinMulhern) [inline db/schema.rb]: Was this something that you added in yourself @wanyamaman? or was it generated somehow? 😄

**Comment by KevinMulhern** [comment]
Thanks for taking the time to put this together @wanyamaman 😄 

It all looks great to me apart from the couple of small issues Arun commented on. Once those are resolved we can merge this in.  If you need any clarification on any of the comments please don't hesitate to let us know.

File: app/views/static_pages/_contributors.html.erb [hunk 3]
```diff
@@ -1,225 +1,160 @@
...
-			</div>
-		</div>
-
-
 	</div>
-</div>
\ No newline at end of file
+	<div class="text-center mt-5">
+		<%= link_to 'View All Contributors',
+													github_link('theodinproject/graphs/contributors'),
```
Comment (arun1595) [inline app/views/static_pages/_contributors.html.erb]: What's with the spacing?

File: db/schema.rb (hunk 1 above)
Comment (wanyamaman) [inline db/schema.rb]: This was auto generated when I rebased with upstream and migrated my db. I'll remove it.

File: app/views/static_pages/contributing.html.erb (hunk 2 above)
Comment (wanyamaman) [inline app/views/static_pages/contributing.html.erb]: Fixed

File: app/views/static_pages/_contributors.html.erb (hunk 3 above)
Comment (wanyamaman) [inline app/views/static_pages/_contributors.html.erb]: Fixed. My text editor wasn't wrapping properly

**Comment by KevinMulhern** [comment]
LGTM 👍

**Pull request merged**